app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -14,7 +14,7 @@ youtube_video:
 navigation: 20.10
 navigation_title: Train to teach in England as a non-UK citizen
 navigation_description: Find out how to train to teach in England as a non-UK citizen or foreign student and get English qualified teacher status (QTS).
-image: "static/images/content/hero-images/0034.jpg"
```

Comment:
This swap-out is fine - but we are still using this 0034 image on quite a few non-uk pages

Would it be possible to switch it out elsewhere in the non-UK section so we are using a different image on each non-UK page?